src/elements/hx-switch/index.js

```diff
@@ -0,0 +1,94 @@
+import { HXFormControlElement } from '../../interfaces/HXFormControlElement/index.js';
```

Comment:
@lalithkarikelli, I modified this component and added the `<hx-switch>` label logic.   We'll discuss these updates.